metricshub-snmp-extension/src/main/java/org/sentrysoftware/metricshub/extension/snmp/SnmpExtension.java

```diff
@@ -107,12 +95,11 @@ public Set<Class<? extends Criterion>> getSupportedCriteria() {
 	}
 
 	@Override
-	public void checkProtocol(TelemetryManager telemetryManager) {
+	public boolean checkProtocol(TelemetryManager telemetryManager) {
 		// Retrieve the hostname
 		final String hostname = telemetryManager.getHostConfiguration().getHostname();
 
-		// Retrieve the host endpoint monitor
-		final Monitor hostMonitor = telemetryManager.getEndpointHostMonitor();
+		log.info("Hostname {} - Performing protocol health check.", hostname);

```

Comment:
`log.info("Hostname {} - Performing {} protocol health check.", hostname, getIdentifier());`

Reply:
Move the `log.info` after the (snmpConfiguration == null) check please.